behat.yml

```diff
@@ -7,23 +7,23 @@ default:
                     adapterId: massive_search.adapter.zend_lucene
             bundle: MassiveSearchBundle
             filters:
-                tags: @zend_lucene
+                tags: '@zend_lucene'
```

Comment:
i have read yesterday that if your are using `'` you dont have to mask everything (except `'`)